src/lib/exeRunner.ts

```diff
@@ -1,93 +1,93 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 import { spawn, spawnSync } from "child_process";
-import os = require('os');
-import path = require("path");
+import os = require("os");
```

Comment:
changing the existing runners here seems wrong. Existing actions will keep using those until switched over, so why "improve" now?

Reply:
A lot of extra noise was generated from me running prettier. The only runner that I've changed now is exeRunner; I've extracted the code for resolving the exe's path into a separate file so that it can be re-used for configuring the cli-wrapper runners.

Previously, I also took the liberty of removing the exeName parameter on the legacy runners and moving the exeName to the end of the path, because that's all it was doing in the code anyways, but I've reverted that change to keep this PR more compact and because that code is ultimately going to be phased out anyways.

Reply:
Edit: also I needed to add a coalescing operator to one of the error codes because it was causing the project to not compile.